apple-developer-accounts.md

```diff
@@ -1,38 +1,10 @@
 # Apple Developer Accounts 
 # 🌟
 
+* Create an Apple ID on your Nodes email
+* Have Service Desk add you to the project you need by making an Access Request (Internal) request here https://nodes-internal.zendesk.com/
+* Remember to ask for access to create certificates and provisioning profiles (there's an extra checkbox they have to check if they add you as Developer)
```

Comment:
This is something we also want to take over in IT/Service Management at some point, so that only very few people are actually able to generate production certs etc. But for now this is good imho :)